pkg/model/labels.go

```diff
@@ -446,3 +449,19 @@ func ParseSessionID(s string) (SessionID, error) {
 	}
 	return SessionID(binary.LittleEndian.Uint64(b[:])), nil
 }
+
+type MappingVersion struct {
+	Repository string `json:"repository,omitempty"`
+	Commit     string `json:"commit,omitempty"`
+}
```

Comment:
I did not understand what you were referring to.

Reply:
I mean if someone renames their repo we will still be using the old name. This is not a problem in vast majority of cases, as VCSs typically handle aliases and redirects well. Problems may appear with on-prem repos and DNS names. This is not an issue as of now as the only VCS we're going to support in v1 is Github.

Reply:
Yeah I think that's fine, address changes but I suspect it minor problem when our blocks right now are only few hours long.

Introducing a new address will also trigger a new mapping. Since AFAIK we hash the mapping ID using the buildID.